contrib/scripts/find-python-unittest

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; specifically version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: Red Hat Inc. 2021
+# Author: Cleber Rosa <cleber@redhat.com>
+
+#
+# Simple script that, given Python file containing unittests, returns the canonical
+# location of each test, each one on its own line.
+#
+# Warning: this script uses Python's unittest TestLoader.discover()
+# and will load (and thus eval) the Python files given in the command
+# as arguments.
+#
+
+import os
+import sys
+import unittest
+
+
+def get_tests(suite):
+    result = []
+    if hasattr(suite, '__iter__'):
+        for s in suite:
```

Comment:
what is s here? `for suite in suite`? `for suite in suites` ? Could you please expand this?

Reply:
The suite can either contain other suites, or tests.  I'm renaming this to `suite_or_test` in v2.